qtconsole/mainwindow.py

```diff
@@ -109,6 +113,17 @@ def set_window_title(self):
         if ok:
             self.setWindowTitle(title)
 
+    def create_tab_with_existing_kernel(self):
+        name, ok = QtGui.QInputDialog.getText(self,
+                                               "Connect to Existing Kernel",
+                                               "Kernel name:")
+        if not ok:
+            return
+        connection_file = os.path.join(jupyter_runtime_dir(), name)
```

Comment:
In the new version it is not used as I use the file dialog to find the connection files. The `jupyter_runtime_dir()` function is now needed to find the actual directory.